src/test/java/com/iexec/core/chain/DealWatcherServiceTests.java

```diff
@@ -56,7 +56,7 @@
 
     @BeforeEach
     public void init() {
```

Comment:
If you like you can `Remove this 'public' modifier.`

Reply:
Sure, let's remove all useless `public` modifiers of tests methods/classes!